staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -136,6 +140,8 @@ func (s *SecureServingOptions) AddDeprecatedFlags(fs *pflag.FlagSet) {
 	fs.MarkDeprecated("public-address-override", "see --bind-address instead.")
 }
 
+// ApplyTo fills up serving information in the server configuration.
+// Note: you might want to take a look at ForceLoopbackConfigUsage method.
```

Comment:
I don't think we need to call attention to it.  I don't think anyone outside of the `kube-apiserver` and federation API server will consider themselves to be the source of truth for kube resources.